addons/website_slides/controllers/main.py

```diff
@@ -669,13 +669,33 @@ def slide_channel_join(self, channel_id):
         success = request.env['slide.channel'].browse(channel_id).action_add_member()
         if not success:
             return {'error': 'join_done'}
+
+        if request.env.user.employee_ids:
+            course = request.env['slide.channel'].browse(channel_id)
+            msg = _('The employee subscribed to the course <a href="%(link)s">%(course)s</a>',
+                link=course.website_url,
+                course=course.name)
+            request.env.user.employee_ids\
+                .filtered(lambda e: e.company_id == request.env.company)\
+                .message_post(body=msg)
```

Comment:
This should be done in a bridge module instead, because you can have website_slides installed without `hr` and it will break. You can hook on the method `_action_add_members()`